packages/nuxeo-web-ui-ftest/pages/ui/browser/document_layout.js

```diff
@@ -1,14 +1,14 @@
 import BasePage from '../../base';
 
 export default class DocumentLayout extends BasePage {
-  getField(field) {
+  getField(field, selector = '') {
     driver.waitForExist(this._selector);
     driver.waitForVisible(this._selector);
-    return this.el.element(`[name="${field}"]`);
+    return this.el.element(`${selector} [name="${field}"]`);
```

Comment:
The only "issue" I can foresee is that when we don't pass a selector we will be searching for: ` [name="..]` (with a white space).
In theory it shouldn't be a big deal, but we can maybe move the ternary to here, to avoid that 🤷🏼‍♂️ 

Just an idea 😅

Reply:
Do you mean doing this check? 
```
    const fieldEl = selector ? this.el.element(`${selector} [name="${field}"]`) : this.el.element(`[name="${field}"]`)
    return fieldEl;
```

Reply:
Or we could do it inside with a ternary or such thing.
```suggestion
    return this.el.element(`${selector ? selector + ' ' : '' } [name="${field}"]`);
```
Anyway, having the white space doesn't seem to be critical and all the tests passed 👍🏼  🙂 
Don't bother to much with the idea 😅